models/issue_indexer.go

```diff
@@ -7,25 +7,60 @@ package models
 import (
```

Comment:
Out of scope for this PR, but I wonder if we really want this functionality in `models/` rather than in `modules/indexer/`. I think we probably need to restructure `models` to make it less of a grab-bag of everything and try to move struct into perhaps a core-models and peripheral/plugin system.

Reply:
I would like to do that, but it will change more files. I prefer to do that on another refactor PRs.